qm-importdisk: add page (#9028)
tldr-pages/tldr: pages/linux/qm-importdisk.md

# qm importdisk

> Import disk image to a VM as unused disk.

> > The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.

Reviewer: ```
> The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.
```
If necessary we can rephrase this line.

> > The image format has to be supported by qemu-img(1) such as raw, qcow2, qed, vdi, vmdk, vhd.

Reviewer: ```suggestion
> The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.
```

> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

> - Import a VMDK disk image to storage called "local-zfs":

Reviewer: ```suggestion
- Import a VMDK disk image to specific storage:
```

> `qm importdisk {{100}} name-of-disk.vmdk local-zfs`

Reviewer: ```suggestion
`qm importdisk {{vm_id}} {{disk.vmdk}} {{local-zfs}}`
```

Reviewer 2: This command requires the path to the vmdk file, right? Is an storage arbitrary name?
```suggestion
`qm importdisk {{100}} {{path/to/disk.vmdk}} {{storage_name}}`
```

> - Import a VMDK disk image to storage called "local-lvm" as qcow2 format:

Reviewer: ```suggestion
- Import a VMDK disk image to specific storage in a specific format:
```

> `qm importdisk {{100}} name-of-disk.vmdk local-lvm --format qcow2`

Reviewer: ```suggestion
`qm importdisk {{vm_id}} {{disk.vmdk}} {{local-lvm}} --format {{qcow2}}`
```

> `qm importdisk {{vm_id}} {{path/to/disk.vmdk}} {{storage_name}} --format {{qcow2 | raw | vmdk}}`

Reviewer 3: ```suggestion
`qm importdisk {{vm_id}} {{path/to/disk.vmdk}} {{storage_name}} --format {{qcow2|raw|vmdk}}`
```

Mandatory fix: [remove extra spaces around pipe symbol: `--format` values doesn't support `"qcow2 "`, `" raw "`, and `" vmdk"` values](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).
